### DISCUSSION TITLE
Is milo compatible with opcda?

### DISCUSSION THREAD

**Discussion by 787070135** [eclipse-milo/milo#1072] [General]
Is milo compatible with opcda?

**Comment by kevinherron** [comment]
No, not compatible with OPC Classic (COM/DCOM) in any way.

**Reply by 787070135** [reply]
Thanks for answering

**Reply by a-belmeskine** [reply]
How to make Eclipse Milo compatible with OPC DA or a Java solution for writing to an OPC DA server?

**Reply by kevinherron** [reply]
@a-belmeskine you don't make it compatible, you have to find another project to use.
